### DISCUSSION TITLE
Achievements section not visible on profile

### DISCUSSION THREAD

**Discussion by Sahad-09** [Schweinepriester/github-profile-achievements#531] [Q&A]
Even after enabling my achievements through settings it still doesn't show on my profile.

**Comment by jarek-pacocha** [comment]
Hey @Sahad-09,
In order for the achievements section to be visible you need to earn your first achievement. I hope this answers your question. Good luck! 👍

**Comment by Rishav-02** [comment]
Hello @Sahad-09  Firstly you have to earn any badge(achievement) like these
- **Pull Shark -**  Where You have to open pull requests that have been merged.
-  **Star Stuck -** Where You have a repository that has many stars.
-  **Yolo -** Where You merge a pull request without code review.


**These three are simple ones so you try to earn these first**

**Hope you clearly understand it 
Good luck**

**Please "mark as answer" my comment.**

**Comment by khatrisagar** [comment]
you have to achieve badge to display it on your profile.
